### ISSUE TITLE
Update to RL9.5 is broken by supporting nginx rpm packages

### ISSUE THREAD

**Issue by dfied** [grommunio/gromox#113]
As stated in the community forum, please update the requirements of nginx-module-vts and nginx-module-brotli:
https://community.grommunio.com/d/1872-update-issues-on-rhel-9/5

The epoch of the nginx packages have been changed in RHEL / Rocky Linux 9.5. So the Requires sections of both packages should be updated:
Requires: (nginx = 1:1.20.1 or nginx = 2:1.20.1)

Current workaround offered by Andrew in the forum:
1. Download the nginx-module-brotli and nginx-module-vts rpms from the [Grommunio repo](https://download.grommunio.com/community/EL9/)
2. Edit the rpm spec files using rpmrebuild -e -p downloadedfile.rpm
3. Replace the line Requires: nginx = 1:1.20.1 with Requires: (nginx = 1:1.20.1 or nginx = 2:1.20.1)
4. Reinstall nginx-module-brotli and nginx-module-vts using the modified rpm files sudo dnf reinstall ~/rpmbuild/RPMS/x86_64/editedfile.rpm
5. Now the installed packages will not conflict with the latest version of NGINX from RHEL/Rocky and updates can be installed as normal with dnf upgrade

**Comment by jengelh** [comment]
nginx-mod-vts-0.2.2-11.1.x86_64 is published.

**Comment by dfied** [comment]
@jengelh Thanks for changing. The dependencies are now correctly fulfilled.
On my RL9.4 box I now have an error during the transaction test:

`Error: Transaction test error:
  file /etc/redis from install of grommunio-common-26.a6f127d-41.1.x86_64 conflicts with file from package redis-6.2.7-1.el9.x86_64
`

Have there been any changes in grommunio-common about the /etc/redis file structure?

**Comment by jengelh** [comment]
that has nothing to do with nginx, though.

**Comment by dfied** [comment]
Yes, that's correct. Shall I open a new issue?

**Comment by robert-scheck** [comment]
> Yes, that's correct. Shall I open a new issue?

Yes, please, because it's a packaging mistake in `grommunio-common`.

**Comment by dfied** [comment]
@robert-scheck Imho, the redis config issue has been already fixed - so I didn't see a need to open a new issue.
